internal/parser/parser/docx_numbering.go

```
@@ -0,0 +1,659 @@
+		}
+		markerEnd++
+	}
+	if markerEnd >= len(line) || line[markerEnd] != ' ' {
+		return "", false
+	}
+	content := strings.TrimSpace(line[markerEnd:])
+	content = strings.Trim(content, "*_~`")
+	return content, true
+}
+
+func appendDOCXNumberedHeadingOutlines(outlines []docxOutline, headings []docxNumberedHeading) []docxOutline {
```

golangci-lint (2.12.2)

[error] func appendDOCXNumberedHeadingOutlines is unused

(unused)